data/bedrock/common/features.json

```diff
@@ -3,6 +3,50 @@
     "name": "newRecipeSchema",
     "description": "New recipe schema",
     "versions": ["1.16_major", "latest"]
+  },
+  {
+    "name": "serializedItemNbtIsExtraData",
+    "description": "in newer versions, NBT fields are within the extra field, but before 1.16.220, they're not",
+    "versions": ["1.16.220", "latest"]
+  },
+  {
+    "name": "spawnEggsUseInternalIdInNbt",
+    "description": "in older versions, spawn eggs have a field in their nbt called 'internalId' which tells what entity they will spawn",
+    "versions": ["1.16_major", "latest"]
+  },
+  {
+    "name": "spawnEggsHaveSpawnedEntityInName",
+    "description": "in newer versions, spawn eggs have the entity they spawn in their name, ex: 'squid_spawn_egg'",
+    "versions": ["1.16.100", "latest"]
```

Comment:
Tests are failing because 1.16.100 isn't supported, what version should I put instead?

Reply:
1.16.201